pkgs/top-level/all-packages.nix

```diff
@@ -9682,6 +9682,10 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) AppKit Cocoa;
   };
 
+  gtk3-x11 = gtk3.override {
+    x11Support = true;
```

Comment:
Can you maybe add a comment that is for macOS where by default gtk3 uses Cocoa (?) instead of x11?

Reply:
If it's Darwin specific, maybe call it `gtk3-darwin` or `gtk3-cocoa`?

Reply:
That sounds confusing. gtk3 uses cocoa by default which makes sense and `gtk3-x11` uses x11 everywhere. Using gtk3-x11 on linux doesn't change anything.

Reply:
Cocoa/x11 support is probably mutual exclusive - cocoa is preferred as it is the native environment. Building gtk3 without x11 support on linux does not make sense.

Reply:
Exactly, I'm not sure if this broke because of the refactor or a recent update but gui support was disabled or actually didn't work before. I couldn't get Cocoa to work and while gvim with X11 isn't very nice/useful on darwin it does enable +clientserver support.